zproject/settings.py

```diff
@@ -736,11 +737,12 @@ def get_secret(key):
 
 app_srcs = JS_SPECS['app']['source_filenames']
 
-PIPELINE_JS = {}  # Now handled in tools/minify-js
-PIPELINE_JS_COMPRESSOR  = None
-
-PIPELINE_CSS_COMPRESSOR = 'pipeline.compressors.yui.YUICompressor'
-PIPELINE_YUI_BINARY     = '/usr/bin/env yui-compressor'
+PIPELINE = {
+    'PIPELINE_ENABLED': PIPELINE,
```

Comment:
This is very confusing, since `PIPELINE` is set above to be the bool for whether the pipeline feature is enabled.  Can you refactor this to something less confusing?  Probably a sane approach would be to first rename the old PIPELINE variable to PIPELINE_ENABLED...